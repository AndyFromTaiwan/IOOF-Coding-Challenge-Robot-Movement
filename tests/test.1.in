PLACE 2,2,NORTH
REPORT
MOVE
REPORT
MOVE
REPORT
MOVE
REPORT

PLACE 2,2,EAST
REPORT
MOVE
REPORT
MOVE
REPORT
MOVE
REPORT

PLACE 2,2,SOUTH
REPORT
MOVE
REPORT
MOVE
REPORT
MOVE
REPORT

PLACE 2,2,WEST
REPORT
MOVE
REPORT
MOVE
REPORT
MOVE
REPORT