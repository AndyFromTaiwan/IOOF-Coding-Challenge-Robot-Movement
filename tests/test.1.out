2,2,NORTH
2,3,NORTH
2,4,NORTH
2,4,NORTH
2,2,EAST
3,2,EAST
4,2,EAST
4,2,EAST
2,2,SOUTH
2,1,SOUTH
2,0,SOUTH
2,0,SOUTH
2,2,WEST
1,2,WEST
0,2,WEST
0,2,WEST